src/scripts/create-user.ts

```diff
@@ -0,0 +1,133 @@
+import { CreateUserRequestOptions } from '@okta/okta-sdk-nodejs';
+import { Argv } from 'yargs';
+import { RootCommand } from '..';
+
+import {
+  oktaManageClient,
+  OktaConfiguration,
+  User,
+  oktaUserAsUser,
+  getUser,
+} from './services/user-service';
+
+import { generatePassword } from './services/password_manager';
+
+const createUser = async (
+  oktaConfiguration: OktaConfiguration,
+  password: string,
+  email: string,
+  firstName = 'Unknown',
+  lastName = 'User'
+): Promise<User> => {
+  const client = oktaManageClient(oktaConfiguration);
+
+  const maybeOktaUser = getUser(email, client);
+  // eslint-disable-next-line functional/no-expression-statement
+  const newUser: CreateUserRequestOptions = {
+    profile: {
+      firstName: firstName,
+      lastName: lastName,
+      email: email,
+      login: email,
+    },
+    credentials: {
+      password: { value: password },
```

Comment:
Does this mean that the password has to be reset on first login?

Reply:
Good question. Following the users documentation ... it seems that it might not be the case. However it needs to be tested with an actual legitimate email address.

Reply:
Please test this to confirm it matches the ticket description.

Reply:
Confirmed match